tx/textual/valuerenderer/bytes.go

```diff
@@ -30,6 +46,9 @@ func (vr bytesValueRenderer) Parse(_ context.Context, screens []Screen) (protore
 	}
 	formatted := screens[0].Text
 
+	// If the formatted string's length is 32 bytes, there's actually no way to
+	// know if it was a hash of even longer bytes, or those bytes as-is. We
+	// simply return the bytes as-is.
```

Comment:
And that's a problem, as above. Detect the hash format and return an error.

Reply:
Fixed, returning an error now with hash prefix.